packages/core/src/RenderPipeline/RenderQueue.ts

```
@@ -30,8 +32,16 @@
   }
 
   sortBatch(compareFunc: Function, batcherManager: BatcherManager): void {
-    this._sort(compareFunc);
-    this._batch(batcherManager);
+    this.sort(compareFunc);
+    this.batch(batcherManager);
+  }
+
+  sort(compareFunc: Function): void {
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)